### PR TITLE
Write README

### PR DESCRIPTION
**Pull request by marnen**
@kytrinyx I'd appreciate your input here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +68,7 @@ Before submitting your pull request, you'll want to verify the changes in two wa
 All the tests for GNU APL exercises can be run from the top level of the repo with
 
 ```
-# add this command
+bin/run-all-tests
 ```
 
 For the Exercism-specific linting, please see [the documentation](https://github.com/exercism/docs/blob/master/language-tracks/configuration/linting.md).
```
Comment (marnen) [inline README.md]: What's the deal with this? The link is broken, and I'm not sure what this is about.

Comment (kytrinyx) [reply README.md]: The docs are a bit of a mess at the moment :/ it's described here: https://github.com/exercism/docs/blob/a8ea5db92e2a2d2839e66ec10c7687b3b7db002a/language-tracks/configuration/configlet.md

**Comment by kytrinyx** [comment]
I'll give this a proper review before Monday.

**Comment by marnen** [comment]
@kytrinyx Thanks!

File: README.md (hunk 1 above)
Comment (marnen) [reply README.md]: OK. I'll look there.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ Exercism exercises in GNU APL.
 
 ## Setup
 
-The simplest way to install GNU APL is ...
+On MacOS, GNU APL is available as a [Homebrew](http://brew.sh) formula; just type `brew install gnu-apl` to install it.
+
+On other operating systems, the simplest way to install GNU APL is probably to follow the instructions on the [GNU APL website](https://www.gnu.org/software/apl/). Note that version 1.7 seems to have build issues on some systems; if you can't build 1.7, then 1.6 should work.
```
Comment (marnen) [inline README.md]: I should probably also mention something about where to find an APL keyboard layout.

**Comment by marnen** [comment]
@kytrinyx Glad you like it! I have a couple more things to add, and then I'll merge it.

File: README.md (hunk 2 above)
Comment (kytrinyx) [reply README.md]: Yes, that would be very handy!

**Pull request merged**